Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/LocalViewBookmarkLoader.cpp

```diff
@@ -266,25 +272,28 @@ namespace AzToolsFramework
                     m_bookmarkfileName = bookmarkComponent->GetLocalBookmarksFileName();
                     using FixedValueString = AZ::SettingsRegistryInterface::FixedValueString;
                     AZStd::string bookmarkKey = ViewBookmarksRegistryPath + m_bookmarkfileName;
-                    auto viewBookmarkSettingsKey = FixedValueString::format("%s",bookmarkKey.c_str());
+                    auto viewBookmarkSettingsKey = FixedValueString::format("%s", bookmarkKey.c_str());
```

Comment:
Why not just use bookmarkKey directly?

The SettingsRegistry::Visit function accepts a string_view, both of which an AZStd::fixed_string and AZStd::string are convertible to.

Reply:
Good shout, this is fixed 👍